### DISCUSSION TITLE
Linear regression training energy

### DISCUSSION THREAD

**Discussion by lammps-vasp** [deepmodeling/deepmd-kit#749] [Q&A]
DP developer, I want to know if demd can be used to train energy alone, which is similar to machine learning. Now I use energy alone for training, and the effect is too poor. As the developer said, [] is equivalent to linear regression. Is there a problem in parameter adjustment, or can't I train with energy alone? Thank you for your advice

Looking forward to your reply!

**Comment by AnguseZhang** [comment]
What's the meaning of training energy alone?

**Reply by lammps-vasp** [reply]
Observe whether the energy can be trained well. The energy data is from MOLPRO

**Reply by AnguseZhang** [reply]
Do you have force data? Training with energy alone may cause overfitting problem.

**Reply by lammps-vasp** [reply]
There is no force data, there is only energy, but there is no over fitting, but the result is very poor, not on the line y = x

**Reply by AnguseZhang** [reply]
Then you should check the quality of training data, including first-principles convergence, data conversion, e.t.c. and traning parameters.
